resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
+      <ComponentRef Id="Service_MARIONETTECOLLECTIVE_svc" />
+    </ComponentGroup>
+
+    <DirectoryRef Id="MARIONETTECOLLECTIVEBINDIR">
+      <Component Id='Service_MARIONETTECOLLECTIVE_svc'
```

Comment:
@melissa That's fine, I can open up a maint PR that overrides the service name to MCO. Which will make all these become MCO instead of MARIONETTECOLLECTIVE. And @jcoconnor overriding the name in install service is already provided. If you pass in install service like so: `install_service #{path_to_bin} nil #{name}` it will actually already do the name override for us =) no need to add anything to install_service. refer to: https://github.com/McdonaldSeanp/vanagon/blob/feature/RE-6455/dirlist/spec/lib/vanagon/platform/windows_spec.rb#L186L197